README.md

```diff
@@ -1,2 +1,5 @@
 ## Color Palette Contrast Analyzer
 
+## Contributing
+
+On Mac OS X, you will likely need watchman to be installed in order to get `yarn test` to run. Use Homebrew to install watchman with `brew install watchman`.
```

Comment:
interesting, we should keep our eyes on https://github.com/facebook/create-react-app/issues/871